### PR TITLE
Tests PR in to develeop branch

### PR DESCRIPTION
**Pull request by devv77**
I think that the tests are done, so maybe we could do the PR into develop.

### CODE REVIEW

**Review by IIB3nceII — APPROVED** [review]

**Review by tsaskater — APPROVED** [review]
image

**Review by belhart — APPROVED** [review]

File: back-end/Models/CommentReacters.cs [hunk 1]
```diff
@@ -4,17 +4,22 @@
...
-            public string CommentReactersID { get; set; }
-            public string CommentID { get; set; }
-            public string UserID { get; set; }
-            public int Choice { get; set; } // TODO: bool inkább? legalább byte?
-      }
+    public class CommentReacters
+    {
+        //[Key]
+        //[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        //public string CommentReactersID { get; set; }
```
Comment (rolandbogdan) [inline back-end/Models/CommentReacters.cs:16]: Why was this table's own id commented out?

File: back-end/Models/SubjectUsers.cs [hunk 2]
```diff
@@ -4,15 +4,21 @@
...
-            [Key]
-            [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-            public string SubjectID { get; set; }
-            public string UserID { get; set; }
-      }
+    public class SubjectUsers
+    {
+        //[Key]
+        //[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        //public string SubjectUsersID { get; set; }
```
Comment (rolandbogdan) [inline back-end/Models/SubjectUsers.cs:16]: Same question here

File: back-end/Models/TopicTags.cs [hunk 3]
```diff
@@ -4,16 +4,22 @@
...
-            [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-            public string TopicID { get; set; }
-            public string TagID { get; set; }
-            public DateTime CreationDate { get; set; }
-      }
+    public class TopicTags
+    {
+        //[Key]
+        //[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        //public string TopicTagsID { get; set; }
```
Comment (rolandbogdan) [inline back-end/Models/TopicTags.cs:16]: And here

**Pull request closed without merging**